.github/workflows/build.yaml

```diff
@@ -23,23 +23,18 @@ jobs:
       uses: actions/checkout@main
 
     - name: Update jupyter dependencies with repo2docker
-      uses: yuvipanda/repo2docker-action@fix-output
+      uses: yuvipanda/repo2docker-action@docker-push
       id: r2d
       with:
         # Make sure username & password/token pair matches your registry credentials
         DOCKER_USERNAME: ${{ secrets.QUAY_USERNAME }}
         DOCKER_PASSWORD: ${{ secrets.QUAY_PASSWORD }}
         DOCKER_REGISTRY: "quay.io"
-        NO_PUSH: "true"
         IMAGE_NAME: "2i2c/utoronto-image"
 
     - name: Login to Quay.io
       uses: docker/login-action@v1
       with:
         registry: quay.io
         username: ${{ secrets.QUAY_USERNAME }}
-        password: ${{ secrets.QUAY_PASSWORD }}
-
-    - name: docker push
-      # FIXME: Due to timeouts in the r2d action, we push manually
-      run: docker push ${{ steps.r2d.outputs.IMAGE_SHA_NAME }}
+        password: ${{ secrets.QUAY_PASSWORD }}
```

Comment:
Since r2d is now pushing stuff using docker push (so it already logged into quay), the "Login to Quay" step at the end is no longer needed, right?